xml/System.Runtime.InteropServices.CustomMarshalers/TypeToTypeInfoMarshaler.xml

```diff
@@ -36,10 +36,10 @@
   </Docs>
   <Members>
     <Member MemberName=".cctor">
-      <MemberSignature Language="C#" Value="public TypeToTypeInfoMarshaler ();" />
+      <MemberSignature Language="C#" Value="static TypeToTypeInfoMarshaler ();" />
```

Comment:
```suggestion
      <MemberSignature Language="C#" Value="public static TypeToTypeInfoMarshaler ();" />
```

Reply:
I was under the impression that a static constructor could not have a `public static` signature?

Reply:
That was my assumption as well. But in the 1.1 assembly, it clearly is public. So the best alternative is to base the syntax on the metadata in the assembly. Otherwise, if it were private, we shouldn't be documenting it at all.